putData.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 set -e
 
+echo Putting data for $1 to wikibase.dev
+
 # Inputs for the script
+# TODO change wikibase.dev for real migration
+# TODO change cluster name for real migration
 NEW_PLATFORM_FREE_DOMAIN_SUFFIX="wikibase.dev"
```

Comment:
Do we want this to be a second (possibly optional) parameter? Or we just hard code change it when we come to do the migration to wikibase.cloud?